3-Register/src/app/layout.tsx

```diff
@@ -0,0 +1,31 @@
+import { Manrope } from "next/font/google";
+import Image from "next/image";
+import "./globals.css";
+
+const manrope = Manrope({
+  subsets: ["latin"],
+  weight: ["300", "400", "500", "600"],
+});
+
+export default function RootLayout({
```

Comment:
The component should have the same name as the file

```suggestion
export default function Layout({
```